techmatters/flex-plugins
# CHI-958: Terraform chatbot module

Primary Reviewer @GPaoloni 

## Description

Migrate the terraform chatbot resources into a module, both because it needs doing but also to demo how we could move other resources into modules as required

### Checklist
- [ X ] Corresponding issue has been opened
- [ N/A ] New tests added
- [ N/A ] Strings are localized



### Related Issues
Story - CHI-958

### Verification steps

Set up a new account with the version of the terraform resources prior to these changes
Update to this version
Do a terraform apply, reviewing the plan - it should NOT be deleting & creating chatbot resources if the refactor block is working correctly.
